Prometheus tuning and Metric Thinning (#565)
kiali/kiali.io: content/en/docs/Configuration/p8s-jaeger-grafana/_index.md

---
title: "Prometheus, Jaeger, Grafana"
description: "Kiali data sources and add-ons."
---

> Prometheus and Jaeger are primary data sources for Kiali. This page describes how to configure Kiali to communicate with these dependencies. Grafana can be used as a visualization add-on. A minimalistic Grafana integration available.

Reviewer: ```suggestion
Prometheus and Jaeger are primary data sources for Kiali. This page describes how to configure Kiali to communicate with these dependencies. Kiali optionally provides a minimalistic integration with Grafana.
```

> Prometheus and Jaeger are primary data sources for Kiali. This page describes how to configure Kiali to communicate with these dependencies. Grafana can be used as a visualization add-on. A minimalistic Grafana integration available.

Reviewer 2: As we saw recently, some users may not be aware that Kiali NEEDS Prometheus in order to graph the topology and to display the metric graphs.

Should we be more forceful here? Rather than say these are "primary" data sources, we should say "necessary" data sources or something like that. Perhaps a warning box here saying something to the effect of "Without Prometheus, Kiali will be unable to display the topology graph and other metrics-related visualizations. Without Jaeger, Kiali will be unable to display anything related to tracing."

Note that I see you call this out in the prometheus page (you say prom is "required") but I'm questioning if we should be equally transparent here in this page.

Author: This isn't new text, just reorganized, but the comment is fair.  I can make it more clear.
